leapp/cli/__init__.py

```diff
@@ -7,14 +7,8 @@
 
 
 @command('')
-@command_opt('debug', is_flag=True, help='Enable debug mode', inherit=True)
-@command_opt('verbose', is_flag=True, help='Enable verbose logging', inherit=True)
-def cli(args):
-    os.environ['LEAPP_DEBUG'] = '1' if args.debug else os.environ.get('LEAPP_DEBUG', '0')
-    if os.environ['LEAPP_DEBUG'] == '1' or args.verbose:
-        os.environ['LEAPP_VERBOSE'] = '1'
-    else:
-        os.environ['LEAPP_VERBOSE'] = os.environ.get('LEAPP_VERBOSE', '0')
+def cli(args): # noqa; pylint: disable=unused-argument
+    pass
```

Comment:
re-raising original post: @vinzenz, do you know how to get rid of this function here? I haven't studied the cli code much but it seems to me it has no meaning here now.

Reply:
@vinzenz, could you please add a comment explaining why this function needs to be here. Thanks. We can just leave this strange code here but it should be at least properly commented.

Reply:
this function used to handle debug and verbose for all sub commands by setting the environment variable, so not every sub command had to implement this.

Reply:
and i am not sure what is so strange about it, it was just a way of not having to do the same stuff every time

Reply:
I understand the original purpose, but now it has lost it. I and Alex were trying to get rid of it but with no success.

Reply:
Alright, we'll keep it this way.

Reply:
> ... with no success.

could you elaborate on that?

Reply:
> could you elaborate on that?
I don't remember what constructs I was trying but basically whatever change I tried to remove the function warranted a traceback. After I gave up for not wielding enough python powers.